wagtail/embeds/finders/instagram.py

```diff
@@ -0,0 +1,84 @@
+import json
+import re
+
+from urllib import request as urllib_request
+from urllib.error import HTTPError, URLError
+from urllib.parse import urlencode
+from urllib.request import Request
+
+from django.conf import settings
+
+from wagtail.embeds.exceptions import EmbedException, EmbedNotFoundException
+
+from .base import EmbedFinder
+
+
+class AccessDeniedInstagramException(EmbedException):
```

Comment:
Maybe also here make it more explicit?

```suggestion
class AccessDeniedInstagramOEmbedException(EmbedException):
```